packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -35,20 +35,20 @@
             <img
               :src="allBrowsersIcons[browser.displayName]"
               :alt="browser.displayName"
-              class="inline w-40px h-40px"
+              class="h-40px w-40px inline"
             >
           </div>
-          <div class="pt-2 text-lg text-indigo-600">{{ browser.displayName }}</div>
-          <div class="text-xs text-gray-400">v{{ browser.majorVersion }}.x</div>
+          <div class="text-lg pt-2 text-indigo-600">{{ browser.displayName }}</div>
+          <div class="text-xs text-gray-400">v{{ browser.majorVersion }}</div>
```

Comment:
We removed the `v` in the other list, did we want to also remove it here?

Reply:
Confirmed with Ryan, he wants the `v` to stay in this case.